src/lib/transformers.js

```diff
@@ -0,0 +1,40 @@
+const colors = require('colors');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
const colors = require("colors");
```